.env.development

```diff
@@ -0,0 +1,2 @@
+REDIS_HOST=<HOST_HERE>
```

Comment:
Instead of `<HOST_HERE>` and so one real values should be used. If this is for vagrant environment then I suggest you point it into host and port provided by the environment.